src/app/modules/item/pages/item-forum/item-forum.component.ts

```diff
@@ -132,8 +153,30 @@ export class ItemForumComponent implements OnInit, OnChanges, OnDestroy {
     this.refresh$.next();
   }
 
+  toggleThread({
+    rowData,
+    itemId,
+    participantId
+  }:{
+      rowData: Thread & { isOpened: boolean, rowIndex: number },
+      itemId: string,
+      participantId: string,
+    }
```

Comment:
In this project, we would write params on one line when it fits on one line, so: 
```suggestion
  toggleThread({ rowData, itemId, participantId }:{
      rowData: Thread & { isOpened: boolean, rowIndex: number },
      itemId: string,
      participantId: string,
    }
```